### DISCUSSION TITLE
Installing GrandOrgue on Linux Mint

### DISCUSSION THREAD

**Discussion by Jean-Gabriel1982** [GrandOrgue/grandorgue#696] [General]
Hi @oleg68 

I've just received my new computer (with 64Go Ram !! :)) and i installed Linux Mint.
As i'm not good at Linux (i'm Windows or MacOs), could you tell me how to install GrandOrgue on Linux ?
I don't know which package to choose (arch64.tar.gz or x86_64.tar.gz, and how to install it after download .. :)

Many thanks !!

**Comment by oleg68** [comment]
Hi, @Jean-Gabriel1982 

Linux mint is a debian-based system. I hope you computer has an intel or an amd cpu (not arm), so you need amd64 files.

You should download three files: [0.3.1.2341-15.os/grandorgue_0.3.1.2341-15.os_amd64.deb](https://github.com/oleg68/GrandOrgue/releases/download/0.3.1.2341-15.os/grandorgue_0.3.1.2341-15.os_amd64.deb), [grandorgue-demo_0.3.1.2341-15.os_all.deb](https://github.com/oleg68/GrandOrgue/releases/download/0.3.1.2341-15.os/grandorgue-demo_0.3.1.2341-15.os_all.deb) and [grandorgue-resources_0.3.1.2341-15.os_all.deb](https://github.com/oleg68/GrandOrgue/releases/download/0.3.1.2341-15.os/grandorgue-resources_0.3.1.2341-15.os_all.deb) to some directory and type the following command:

```
apt-get install ./grandorgue_0.3.1.2341-15.os_amd64.deb ./grandorgue-resources_0.3.1.2341-15.os_all.deb ./grandorgue-demo_0.3.1.2341-15.os_all.deb
```

See more in [INSTALL.md](https://github.com/oleg68/GrandOrgue/blob/master/INSTALL.md)

**Reply by Jean-Gabriel1982** [reply]
Hi @oleg68 

thank you very much !
Now it does not work but I will see INSTALL.md :-)
